$OpenBSD$

Remove warp/warp-tls source-repository-packages (Reaper API change).

--- cabal.project.orig
+++ cabal.project
@@ -59,16 +59,3 @@
     location: https://github.com/simplex-chat/zip.git
     tag: 2eff156c3aac389e35d38bf10a52733d7061640a

--- waiting for published warp-tls-3.4.7
-source-repository-package
-    type: git
-    location: https://github.com/yesodweb/wai.git
-    tag: ec5e017d896a78e787a5acea62b37a4e677dec2e
-    subdir: warp-tls
-
--- backported fork due http-5.0
-source-repository-package
-    type: git
-    location: https://github.com/simplex-chat/wai.git
-    tag: 2f6e5aa5f05ba9140ac99e195ee647b4f7d926b0
-    subdir: warp
